scripts/sel4.sh

```diff
@@ -86,6 +86,7 @@ as_root apt-get install -y --no-install-recommends -t bullseye \
     gcc-8-arm-linux-gnueabihf \
     gcc-8-base \
     gcc-8-multilib \
+    gcc-riscv64-unknown-elf \
```

Comment:
We should add a RISC-V QEMU then also, unfortunately there is no `qemu-system-riscv`, but it's still all in `qemu-system-misc`. See https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=qemu-system

Reply:
Right, I forgot about Qemu. And it's worse than that, because I'm having trouble running sel4test with the versions of `qemu-system-riscv64` and `qemu-system-riscv32` provided in that package: `QEMU emulator version 3.1.0 (Debian 1:3.1+dfsg-8+deb10u8`.

They don't support `-machine spike` or `-cpu rv64`, but want you to pick a version (spike_v1.10 or spike_v1.9.1), and no matter which version I pick, sel4test fails.

I might look at building only qemu from source in the riscv toolchain, so that at least that one is current.

Reply:
Hm, this might not actually be broken (although old). qemu for riscv in the current image also doesn't work for me (on the Mac). Somewhat disappointing docker-wise. Might have to have a look at this under Linux.

Reply:
Just logging here that the old QEMU version also does not seem to work under docker in Linux.